src/main/java/com/wavefront/sdk/common/Clock.java

```diff
@@ -0,0 +1,12 @@
+package com.wavefront.sdk.common;
+
+public abstract class Clock {
+    static long currTime = 0;
```

Comment:
is this only for the test? (since I see it being used only in tests)

Reply:
Yes, this is only for tests. For real, we'll just delegate to `System.nanoTime()`.